lib/active_model_serializers/key_transform.rb

```diff
@@ -11,6 +11,7 @@ module KeyTransform
     # @see {https://github.com/rails/rails/blob/master/activesupport/lib/active_support/inflector/methods.rb#L66-L76 ActiveSupport::Inflector.camelize}
     def camel(value)
       case value
+      when Array then value.map { |item| camel(item) }
```

Comment:
This really should have been here the whole time anyway. Welcome change!

@bf4 do you know if there is much of performance improvement that could be made be re-ordering these? or would it not be worth it?